code/game/gamemodes/cult/ritual.dm

```diff
@@ -333,13 +332,25 @@ var/global/list/rnwords = list("ire","ego","nahlizet","certum","veri","jatkaa","
 		var/C = 0
 		for(var/obj/effect/rune/N in rune_list)
 			C++
+
 		if (!istype(user.loc,/turf))
-			user << "<span class='warning'>You do not have enough space to write a proper rune.</span>"
+			to_chat(user, "<span class='warning'>You do not have enough space to write a proper rune.</span>")
+			return
+
+		var/turf/T = get_turf(user)
+

```

Comment:
I would add the admin-z check here aswell

Reply:
My original plan was just to stop people from teleporting, you can still have someone write a rune in an admin z-level, like a merchant cultists sending a message or something.